packages/table/preview/index.tsx

```diff
@@ -113,8 +113,11 @@ class FormatsTable extends React.Component<{}, {}> {
     private strings = Utils.times(FormatsTable.ROWS, () => ("ABC " + (Math.random() * 10000)));
 
     public render() {
+
+        const saveTable = (table: any) => (window as any).formatsTable = table;
```

Comment:
This is just for testing purposes, it'll be replaced with an actual button later, I just wanted eyes on it.